app/routes/pages/components/PageDetail.tsx

```diff
@@ -202,3 +92,300 @@ export const GroupRenderer: PageRenderer = ({ page }) => {
+    ],
+  },
+  bedrifter: {
+    title: 'Bedrifter',
+    section: 'bedrifter',
+    pageSelector: selectFlatpageForPages,
+    hierarchySectionSelector: selectPagesForHierarchy('bedrifter'),
+    PageRenderer: FlatpageRenderer,
+    fetchItemActions: [fetchPage],
+  },
+  arrangementer: {
+    title: 'Arrangementer',
+    section: 'arrangementer',
+    pageSelector: selectFlatpageForPages,
+    hierarchySectionSelector: selectPagesForHierarchy('arrangementer'),
+    PageRenderer: FlatpageRenderer,
+    fetchItemActions: [fetchPage],
+  },
+  komiteer: {
+    title: 'Komiteer',
+    section: 'komiteer',
+    pageSelector: selectCommitteeForPages,
+    hierarchySectionSelector: selectCommitteeForHierarchy,
+    PageRenderer: GroupRenderer,
+    fetchAll: () => fetchAllWithType(GroupType.Committee),
+    fetchItemActions: [
+      fetchGroup,
+      (groupId: number) => fetchAllMemberships(groupId, true),
+    ],
+  },
+  revy: {
+    title: 'Revy',
+    section: 'revy',
+    pageSelector: selectCommitteeForPages,
+    hierarchySectionSelector: selectRevueForHierarchy,
+    PageRenderer: GroupRenderer,
+    fetchAll: () => fetchAllWithType(GroupType.Revue),
+    fetchItemActions: [
+      fetchGroup,
+      (groupId: number) => fetchAllMemberships(groupId, true),
+    ],
+  },
+  grupper: {
+    title: 'Grupper',
+    section: 'grupper',
+    pageSelector: selectFlatpageForPages,
+    hierarchySectionSelector: selectPagesForHierarchy('grupper'),
+    PageRenderer: FlatpageRenderer,
+    fetchItemActions: [fetchPage],
+  },
+  utnevnelser: {
+    title: 'Utnevnelser',
+    section: 'utnevnelser',
+    pageSelector: selectFlatpageForPages,
+    hierarchySectionSelector: selectPagesForHierarchy('utnevnelser'),
+    PageRenderer: FlatpageRenderer,
+    fetchAll: fetchAllPages,
+    fetchItemActions: [fetchPage],
+  },
+  personvern: {
+    title: 'Personvern',
+    section: 'personvern',
+    pageSelector: selectFlatpageForPages,
+    hierarchySectionSelector: selectPagesForHierarchy('personvern'),
+    PageRenderer: FlatpageRenderer,
+    fetchItemActions: [fetchPage],
+  },
+  'info-om-abakus': {
+    title: 'Info om Abakus',
+    section: 'info-om-abakus',
+    pageSelector: selectInfoPageForPages,
+    hierarchySectionSelector: () => ({
+      title: 'hehe',
+      items: [],
+    }),
+    PageRenderer: LandingPage,
+    fetchItemActions: [],
+  },
+};
+
+export const categoryOptions = Object.keys(sections)
+  .map<Entry>((key) => sections[key])
+  .filter((entry: Entry) => entry.pageSelector === selectFlatpageForPages)
+  .map<{
+    value: string;
+    label: string;
+  }>((entry: Entry) => ({
+    value: entry.section,
+    label: entry.title,
+  }));
+
+const getSection = (sectionName) =>
+  sections[sectionName] || {
+    pageSelector: selectNotFoundpageForPages,
+    PageRenderer: HTTPError,
+    fetchItemActions: [],
+  };
+
+const loadData = async (pageSlug, section, loggedIn, dispatch) => {
```

Comment:
It would be great to have this function typed, but I suppose that might not be first priority right now.